web-common/src/features/dashboards/dimension-table/DimensionHeader.svelte

```diff
@@ -18,12 +18,24 @@
   import Spinner from "../../entity-management/Spinner.svelte";
   import { metricsExplorerStore } from "web-common/src/features/dashboards/stores/dashboard-stores";
   import ExportDimensionTableDataButton from "./ExportDimensionTableDataButton.svelte";
+  import { getStateManagers } from "../state-managers/state-managers";
+  import { SortType } from "../proto-state/derived-types";
 
   export let metricViewName: string;
   export let dimensionName: string;
   export let isFetching: boolean;
   export let excludeMode = false;
 
+  const stateManagers = getStateManagers();
+  const {
+    selectors: {
+      sorting: { sortedByDimensionValue },
+    },
+    actions: {
+      sorting: { toggleSort },
+    },
+  } = stateManagers;
```

Comment:
Note that this pattern allows you to easily search through existing actions and selectors using autocomplete, with full type-hinting and doc comments.